apps/emqx_durable_storage/src/emqx_ds_replication_layer.erl

```diff
@@ -43,30 +43,41 @@
 %% Type declarations
 %%================================================================================
 
+%% # "Record" integer keys.  We use maps with integer keys to avoid persisting and sending
+%% records over the wire.
+
+%% tags:
+-define(stream, stream).
+-define(it, it).
```

Comment:
Again, I'm slightly late to the party, but: why do we need tags? There should be no places where we send or expect either stream or iterator (although, admittedly, they could help if this happens accidentally). Given that they are atoms, we're kinda back to that problem where we spend non-negligible portion of bandwidth on something that never changes, though this is arguably a minor issue right now.

Reply:
True.  As you mentioned, it would help to catch accidents.  Maybe the tag value itself could be an integer, then that would also resolve the issue, right?